homework_1/task1/fizzbuzz/fizzBuzz.go

```diff
@@ -0,0 +1,40 @@
+package fizzbuzz
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+func FizzBuzz(start, end int) ([]string, error){
+	if start > end {
+		return nil, errors.New("range start is greater than range end")
+	}
+
+	if start <= 0 {
+		return nil, errors.New("start is 0 or negative")
+	}
+
+	if end <= 0 {
+		return nil, errors.New("end is 0 or negative")
+	}
```

Comment:
Considering the first 2 checks, this check is redundant (it's not possible for end to be negative).